sticky-react/src/app/components/header/Header.js

```diff
@@ -4,70 +4,162 @@ import {
     AppBar,
     Toolbar,
     IconButton,
-    Typography
+    Typography,
+    Button,
+    Menu
 } from '@material-ui/core';
 import MenuIcon from '@material-ui/icons/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
 import Icon from '@material-ui/core/Icon';
 
 //logo
 import logo from '../../../assets/logo_name.svg'
 
-
 class Header extends React.Component {
 
+    constructor(props) {
+        super(props)
+        
+        this.userDialogOpen = false
+        this.handleUserMenuClick = this.handleUserMenuClick.bind(this)
+        this.handleQuitBtnClick = this.handleQuitBtnClick.bind(this)
+  
+        this.menu = [
+            {
+                title:"Sair",
```

Comment:
Also, can we Localize the Strings of our application?

If not, please open an issue so we can track and work on adding localization later on.